### PR TITLE
Document how to turn off Piwik and bug reports (#6738)

### LINKED ISSUE #6738: Document how to turn off piwik and bug reports

### PR DESCRIPTION
**Pull request by aaronraimist**
Signed-off-by: Aaron Raimist <aaron@raim.ist>

Fixes #6738

### CODE REVIEW

**Comment by turt2live** [comment]
Fixes #6738

**Comment by aaronraimist** [comment]
I don't know if that works in comments but I added it to the main PR description

File: README.md [hunk 1]
```diff
@@ -134,7 +134,8 @@ For a good example, see https://riot.im/develop/config.json
 1. `integrations_rest_url`: URL to the REST interface for the integrations server.
 1. `integrations_widgets_urls`: list of URLs to the REST interface for the widget integrations server.
 1. `bug_report_endpoint_url`: endpoint to send bug reports to (must be running a
-   https://github.com/matrix-org/rageshake server)
+   https://github.com/matrix-org/rageshake server). Bug reports can be disabled by leaving the 
+   `bug_report_endpoint_url` out of your config file.
```
Comment (turt2live) [inline README.md]: I wonder if we should add some sort of text here to say that bug reports aren't automatic.

Comment (turt2live) [reply README.md]: Something like "Bug reports are sent when user click "Send Logs" within the application" is probably okay.

**Pull request merged**